database/step_test.go

```diff
@@ -176,7 +182,7 @@ func TestDatabase_Step_Validate_NoNumber(t *testing.T) {
 		ID:      sql.NullInt64{Int64: 1, Valid: true},
 		BuildID: sql.NullInt64{Int64: 1, Valid: true},
 		RepoID:  sql.NullInt64{Int64: 1, Valid: true},
-		Name:    sql.NullString{String: "foo", Valid: true},
```

Comment:
I believe this was removed in error 👍 

I recommend we keep the `Name` field here and add the `Image` field as well